src/argilla_sdk/records/_export/_generic.py

```diff
@@ -60,24 +60,24 @@ def _export_to_dict(
         if orient == "names":
             dataset_records: dict = defaultdict(list)
             for record in records:
-                for key, value in self.__record_to_dict(record=record, flatten=flatten).items():
+                for key, value in self._record_to_dict(record=record, flatten=flatten).items():
                     dataset_records[key].append(value)
         elif orient == "index":
             dataset_records: dict = {}
             for record in records:
-                dataset_records[record.external_id] = self.__record_to_dict(record=record, flatten=flatten)
+                dataset_records[record.external_id] = self._record_to_dict(record=record, flatten=flatten)
         else:
             raise ValueError(f"Invalid value for orient parameter: {orient}")
-        return dataset_records
+        return dict(dataset_records)
```

Comment:
dataset_records is not already a dictionary?

Reply:
It's a `defaultdict`.

Reply:
ahh got it. Similar but not the same. Thanks!